src/fight.cpp

```diff
@@ -3562,7 +3562,7 @@ bool raw_damage(struct char_data *ch, struct char_data *victim, int dam, int att
       break;
   }
 
-  if (GET_MENTAL(victim) < 100 || GET_PHYSICAL(victim) < 0)
+  if (GET_MENTAL(victim) < 100 || GET_PHYSICAL(victim) < 1)
```

Comment:
GET_MENTAL and GET_PHYSICAL are both multiplied by 100 to simulate two digits of precision, so this change takes it from a physical value of 0.00 to a physical of 0.01. It'd probably work best if it was < 100 like mental, since the game should put the victim in a mortally wounded state at that point.

Reply:
But then wouldn't any instance of damage trigger the if statement? Because < 100 would also trigger after 1 box of damage right? Or is `if (FIGHTING(ch) == victim)` only true if the target is morted anyways?

Reply:
`(int) (GET_PHYSICAL(victim)/100)` is the character's current number of physical boxes (that is, it starts at 1000 for an unwounded character with max 10 physical boxes and is reduced as they take damage).

`if (FIGHTING(ch) == victim)` checks to make sure that `victim` is `ch`'s current target.

Reply:
Oh wait I completely misread what Lucien said. I think `GET_PHYSICAL(victim) < 100` would work then since it'd trigger when they go below 1 box of physical. Thanks for clearing stuff up!